libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginEmailFragment.java

```diff
@@ -369,6 +369,7 @@ protected void next(String email) {
             startProgress();
             mRequestedEmail = email;
             mDispatcher.dispatch(AccountActionBuilder.newIsAvailableEmailAction(email));
+            mAnalyticsListener.trackEmailFilled();
```

Comment:
You're right, this event feels out of place. I've only added it because it was in the specifications but it doesn't really make sense. I'll remove it and replace it with a generic interaction/click event in another PR.